code/game/objects/items/storage/boxes.dm

```diff
@@ -318,6 +318,11 @@
 	if(!length(contents))
 		icon_state += "_e"
 
+/obj/item/storage/box/m94/cas
+	name = "\improper M50 CFDP signal pack"
+	desc = "A packet of five M40 CFPD signal Flares. Used to mark locations for fire support. Can be launched from an underslung grenade launcher."
```

Comment:
I'm fairly certain it will be a pack of 7, but please do correct me if it's not.

Reply:
yep its 7 ill fix it in both spots